doc/source/ray-core/objects/serialization.rst

```diff
@@ -23,6 +23,23 @@ Plasma is used to efficiently transfer objects across different processes and di
 
 Each node has its own object store. When data is put into the object store, it does not get automatically broadcasted to other nodes. Data remains local to the writer until requested by another task or actor on another node.
 
+Object Refs
+~~~~~~~~~~~~~~~~~~~
+
```

Comment:
Can you add some notes about usage:

- This should be used as a last resort. Passing ObjectRefs through task args/return values is the recommended approach.
- The deserializing process needs to be part of the same Ray cluster (and I think the same job? Not sure about that part)

Reply:
Sounds good, I added these to the explanation.